content/en/docs/concepts/signals/baggage.md

```diff
@@ -52,23 +55,37 @@ then this risk may not apply, but keep in mind that downstream services could
 propagate Baggage outside your network.
 
 Also, there are no built-in integrity checks to ensure that Baggage items are
-yours, so exercise caution when retrieving them.
+yours, so exercise caution when reading them.
 
-## Baggage is not the same as Span attributes
+## Baggage is not the same as attributes
 
-One important thing to note about Baggage is that it is not a subset of the
-[Span Attributes](/docs/concepts/signals/traces/#attributes). When you add
-something as Baggage, it does not automatically end up on the Attributes of the
-child system’s spans. You must explicitly take something out of Baggage and
-append it as Attributes.
+An important thing to note about baggage is that it is a separate key-value
+store and is unassociated with attributes on spans, metrics, or logs.
 
-For example, in .NET you might do this:
+To add Baggage entries to attributes, you need to explicitly read that data and
+add it. For example, in .NET you might do this:
 
 ```csharp
 var accountId = Baggage.GetBaggage("AccountId");
 Activity.Current?.SetTag("AccountId", accountId);
 ```
 
+Because one of the most common use cases for Baggage is to add data to
+[Span Attributes](/docs/concepts/signals/traces/#attributes) across a whole
+trace, several languages have Baggage Span Processors that automatically read
```

Comment:
could we add links to them? Or mention something like "consult language specific doc to learn how to use them".

Reply:
I decided against this right now since we don't have docs in the languages on using baggage APIs yet. If/when we do, we should absolutely link them though.